radix-engine-interface/src/blueprints/package/invocations.rs

```diff
@@ -16,14 +16,14 @@ pub const PACKAGE_PUBLISH_WASM_IDENT: &str = "publish_wasm";
 #[derive(Debug, Clone, Eq, PartialEq, ScryptoSbor, ManifestSbor)]
 pub struct PackagePublishWasmInput {
     pub code: Vec<u8>,
-    pub definition: PackageDefinition,
+    pub setup: PackageSetup,
```

Comment:
Prefer to have `PackageDefinition`.

Reply:
Will keep this for now as I have a separate PR which updates all naming.